lib/aliquot/validator.rb

```diff
@@ -145,7 +137,7 @@ def errors_formatted(node = [errors])
 
     class Token
       include InstanceMethods
-      class Error < StandardError; end
+      class Error < ::Aliquot::Validation::Error; end
```

Comment:
Not sure what `::Aliquot::Validation::Error` means :)
Specifically the leading `::`.
But I assume you mean `Validator`.

I think I'd subclass them from `Aliquot::Error`. Maybe just create `Aliquot::ValidationError`.

Reply:
`::` is for namespace resolution. A leading `::` specifies it "absolute" rather than "relative" (which is the default). Since we're inside the `Token` class and would like to step into the `Validation` class it's just safer to specify it as an "absolute path".

Where it's placed and what it's called ... I haven't cared about that, just the "user functionality" part of it 🙂